src/components/TradeWidget/TxMessage.tsx

```diff
@@ -30,25 +31,49 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken })
 
   return (
     <TxMessageWrapper>
-      <p>
-        Your are selling {sellTokenAmount} {displaySellToken} for {displayReceiveToken}
-      </p>
-      <p>At a price of</p>
-      <p>
-        {priceInverse} {displaySellToken} per {displayReceiveToken}
-      </p>
-      <p>
-        {price} {displayReceiveToken} per {displaySellToken}
-      </p>
-      <p>
-        <strong>
-          You will receive at least {receiveTokenAmount} {displayReceiveToken}
-        </strong>
-      </p>
-      <p>
-        Your order starts {formatTimeInHours(validFrom || 0, 'now')}{' '}
-        {validUntil && `and will expire ${formatTimeInHours(validUntil, 'Never')}`}
-      </p>
+      <div className="intro-text">
+        <div>Carefully review the information below to make sure everything looks correct.</div>
+      </div>
+      <div className="message">
+        {/* Details */}
+        <div>
+          <strong>Order Details</strong>
+        </div>
+        <div>
+          Sell: <span>{sellTokenAmount}</span> <strong>{displaySellToken}</strong> for{' '}
+          <strong>{displayReceiveToken}</strong>
+        </div>
+        <div>
+          Receive: <UnderlinedText>at least</UnderlinedText> {receiveTokenAmount} <strong>{displayReceiveToken}</strong>
+        </div>
+
+        {/* Prices */}
+        <p>
+          <div>
+            <strong>Prices</strong>
+          </div>
+          <div>
+            Direct: <span>{priceInverse}</span> <strong>{displaySellToken}</strong> per{' '}
+            <strong>{displayReceiveToken}</strong>
+          </div>
+          <div>
+            Inverse: <span>{price}</span> <strong>{displayReceiveToken}</strong> per <strong>{displaySellToken}</strong>
+          </div>
+        </p>
+
+        {/* Order Validity */}
+        <p>
+          <div>
+            <strong>Order Validity Details</strong>
+          </div>
+          <div>
+            Starts: <span>{formatTimeInHours(validFrom || 0, 'Now')}</span>
+          </div>
+          <div>
+            Expires: <span>{formatTimeInHours(validUntil || 0, 'Never')}</span>
+          </div>
```

Comment:
Maybe it's a good idea to add a link here to https://docs.gnosis.io/protocol/docs/intro-batches/#orders

Proposal:
```
⚠️ Learn more about the validity of orders in Batch Exchange [here].
```